database.cc

```diff
@@ -906,7 +906,7 @@ keyspace::make_column_family_config(const schema& s, const database& db) const {
     cfg.enable_disk_reads = _config.enable_disk_reads;
     cfg.enable_disk_writes = _config.enable_disk_writes;
     cfg.enable_commitlog = _config.enable_commitlog;
-    cfg.enable_cache = _config.enable_cache;
+    cfg.enable_cache = _config.enable_cache && s.caching_options().enabled();
```

Comment:
What about ALTER TABLE? We need to be able to change this dynamically.

Reply:
Alter table works as well. I checked.

Reply:
Well - I checked that the property was set, I will double-check that new operations on the table start/stop using cache.

Reply:
You're right. The alter table does not change the behavior dynamically. Only after a restart. I will fix that.